py/torch_tensorrt/dynamo/conversion/impl/attention.py

```diff
@@ -46,6 +51,17 @@ def scaled_dot_product_attention(
             mm,
             scale,
         )
+
+    if is_causal:
```

Comment:
This doesn't support dynamic shapes right ?

Reply:
The current implementation does not, no